frontend/packages/employee-frontend/src/components/person-profile/PersonIncome.tsx

```diff
@@ -54,6 +58,25 @@ const PersonIncome = React.memo(function PersonIncome({ id, open }: Props) {
   }
 
   useEffect(loadData, [id, setIncomes])
+  useEffect(() => {
+    if (incomeDataLoaded) {
```

Comment:
Usually we have used `isSuccess(incomes)` for the two first `if`'s here, wouldn't that do it?

Reply:
Refactored this a bit